pages/api/download/[platform].ts

```diff
@@ -0,0 +1,87 @@
+import { NextApiRequest, NextApiResponse } from "next";
```

Comment:
are square braces some special syntax?

Reply:
square braces in the file name rather.

Reply:
@majestrate Yeah it's a special syntax for NextJS. https://nextjs.org/docs/routing/dynamic-routes